blscosi/bdnproto/struct.go

```diff
@@ -0,0 +1,67 @@
+package bdnproto
+
+import (
+	"errors"
+	"fmt"
+
+	"go.dedis.ch/cothority/v3/blscosi/protocol"
+	"go.dedis.ch/kyber/v3"
+	"go.dedis.ch/kyber/v3/pairing"
+	"go.dedis.ch/kyber/v3/sign"
+	"go.dedis.ch/kyber/v3/sign/bdn"
+	"go.dedis.ch/kyber/v3/sign/cosi"
+)
+
+// BdnSignature is a signature that must be verified using coefficients
+// derived from the public keys
+type BdnSignature []byte
+
+// GetMask returns the bytes representing the mask
+func (sig BdnSignature) GetMask(suite pairing.Suite, pubkeys []kyber.Point) (*sign.Mask, error) {
+	return protocol.BlsSignature(sig).GetMask(suite, pubkeys)
+}
+
+// Point returns the point associated with the signature
+func (sig BdnSignature) Point(suite pairing.Suite) (kyber.Point, error) {
+	return protocol.BlsSignature(sig).Point(suite)
+}
+
+// Verify returns an error if the signature can't be verified or nil if it matches
```

Comment:
Can you mention in the doc string what the threshold policy will be for the verification, please?